src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -0,0 +1,293 @@
+import { useSettings } from "context/useSettings";
+import EditIdentityGroupsBtn from "./actions/EditIdentityGroupsBtn";
+import usePanelParams, { panels } from "util/usePanelParams";
+import CustomLayout from "components/CustomLayout";
+import PageHeader from "components/PageHeader";
+import NotificationRow from "components/NotificationRow";
+import HelpLink from "components/HelpLink";
+import { useDocs } from "context/useDocs";
+import EditIdentityGroupsPanel from "./panels/EditIdentityGroupsPanel";
+import { pluralize } from "util/instanceBulkActions";
+import Tag from "components/Tag";
+
+const PermissionIdentities: FC = () => {
+  const notify = useNotify();
+  const {
+    data: identities = [],
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [queryKeys.identities],
+    queryFn: () => fetchIdentities(),
+  });
+  const { data: settings } = useSettings();
+  const docBaseLink = useDocs();
+  const panelParams = usePanelParams();
+  const [searchParams] = useSearchParams();
+  const [selectedIdentityIds, setSelectedIdentityIds] = useState<string[]>([]);
+
+  if (error) {
+    notify.failure("Loading identities failed", error);
+  }
+
+  const headers = [
+    { content: "Name", className: "name", sortKey: "name" },
+    { content: "ID", sortKey: "id" },
+    { content: "Auth method", sortKey: "authmethod" },
+    { content: "Type", sortKey: "type" },
+    {
+      content: "Groups",
+      sortKey: "groups",
+      className: "u-align--right",
+    },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const filters: PermissionIdentitiesFilterType = {
+    queries: searchParams.getAll(QUERY),
+    authMethod: searchParams.getAll(AUTH_METHOD),
+  };
+
+  const filteredIdentities = identities.filter((identity) => {
+    if (
+      !filters.queries.every(
+        (q) =>
+          identity.name.toLowerCase().includes(q) ||
+          identity.id.toLowerCase().includes(q),
+      )
+    ) {
+      return false;
+    }
+
+    if (
+      filters.authMethod.length > 0 &&
+      !filters.authMethod.includes(identity.authentication_method)
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+
+  const selectedIdentities = identities.filter((identity) => {
+    if (panelParams.identity) {
+      return identity.id === panelParams.identity;
+    }
+
+    return selectedIdentityIds.includes(identity.id);
+  });
+
+  const identitiesForRows =
+    panelParams.panel === panels.identityGroups
+      ? selectedIdentities
+      : filteredIdentities;
+
+  const rows = identitiesForRows.map((identity) => {
+    const isLoggedInIdentity = settings?.auth_user_name === identity.id;
+    const isTlsIdentity = identity.authentication_method === "tls";
+    return {
+      name: isTlsIdentity ? "" : identity.id,
+      key: identity.id,
+      className: "u-row",
+      columns: [
+        {
+          content: (
+            <>
+              {identity.name} <Tag show={isLoggedInIdentity}>You</Tag>
```

Comment:
Design question: A bit more left margin 0.5 rem should look better IMHO.

Reply:
I agree, going to make the change and show it to @piperdeck.